src/services/projects.service.ts

```diff
@@ -76,6 +76,19 @@ export class ProjectsService {
 		}
 	}
 
+	static async setResources(
```

Comment:
wasn't there one like this before?

Reply:
We used it directly from the GRPC `projectClient` in the `build` method, now we need to use it outside the service file, so I added a separate function.
[Refactored](https://github.com/autokitteh/vscode-extension/pull/404/commits/6ee394abe6ce754eabb4067f491c8cc22ee515a0) the reference in the `build` function to use the static function from the service class.